### PR TITLE
VIBES: new port

### PR DESCRIPTION
**Pull request by adarshp**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.6 18G103
Xcode 11.2 11B52

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
Can you please check the azure build logs, as the builds are failing during configure.

```
  install TARGETS given target "Vibes" which does not exist.
-- Configuring incomplete, errors occurred!
```

File: graphics/VIBES/Portfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                    from a lot of programming languages, without complex installation and library\
+                    dependencies. The main design goal of VIBes is to be cross-platform, available\
+                    from different programming languages, simple to set-up, easy to port to a new\
+                    language.
+
+checksums           rmd160  165c8e17970ba55ed78952b674286099cbfd1c68 \
+                    sha256  ff554b0ea18f8c41190894944f614552925ee1c7d94e4133f7e42c0eb864dcda \
+                    size    669976
+
+depends_build       port:qt5\
```
Comment (mf2k) [inline graphics/VIBES/Portfile]: If this port uses qt5, then it should be using the qt5 portgroup to build.

Comment (mf2k) [reply graphics/VIBES/Portfile]: Also, use `depends_build-append` instead to not overwrite portgroup dependencies.

**Comment by adarshp** [comment]
It's failing with Xcode 9.4 with this error:

```
Error: Failed to fetch libarchive: Could not resolve host: distfiles.macports.org
```

Anything I can do on my end to fix this?

File: graphics/VIBES/Portfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                    dependencies. The main design goal of VIBes is to be cross-platform, available\
+                    from different programming languages, simple to set-up, easy to port to a new\
+                    language.
+
+checksums           rmd160  165c8e17970ba55ed78952b674286099cbfd1c68 \
+                    sha256  ff554b0ea18f8c41190894944f614552925ee1c7d94e4133f7e42c0eb864dcda \
+                    size    669976
+
+depends_build-append \
+                    port:qt5-qtsvg\
```
Comment (mf2k) [inline graphics/VIBES/Portfile]: That's not the proper way to add dependencies with the qt5 portgroup. You need to use the `qt5.depends_component` command. See other portfiles (such as `stellarium`) and the qt5 portgroup itself for details.

File: graphics/VIBES/Portfile [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                    port:pbzip2\
+                    port:ibex-lib
```
Comment (mf2k) [inline graphics/VIBES/Portfile]: Please sort all dependencies alphabetically.

File: graphics/VIBES/Portfile (hunk 2 above)
Comment (adarshp) [reply graphics/VIBES/Portfile]: Got it - I have changed it.

File: graphics/VIBES/Portfile (hunk 3 above)
Comment (adarshp) [reply graphics/VIBES/Portfile]: Done!

**Pull request merged**